gravitee-apim-console-webui/src/management/api/creation/v4/services/api-creation-stepper.service.ts

```diff
@@ -94,10 +95,14 @@ export class ApiCreationStepperService {
    * Add a secondary step after the current step.
    */
   public addSecondaryStep(step: NewSecondaryApiCreationStep) {
+    if (this.steps.find(({ id }) => id === step.id)) {
```

Comment:
🤔 I wonder if we could just calculate the generateStepId id before and check if it exists

Reply:
The problem with the fn generateStepId, is that we increment according to the label

Reply:
haha right